pkgs/by-name/dp/dprint/package.nix

```diff
@@ -24,7 +24,12 @@ rustPlatform.buildRustPackage (finalAttrs: {
     hash = "sha256-Lt6CzSzppu5ULhzYN5FTCWtWK3AA4/8jRzXgQkU4Tco=";
   };
 
-  cargoHash = "sha256-1opQaR3vbm/DpDY5oQ1VgA4nf0nCBknxfgOSPZQbtV4=";
+  cargoPatches = [
+    # Upgrade wasmer to 6.1.0-rc.3 to fix build failure with Rust ≥ 1.89.0
+    ./upgrade-wasmer.patch
```

Comment:
A comment linking to the pr here would be nice.

Reply:
Added.